### ISSUE TITLE
Bitrise: Debug/Test Package Deploy

### ISSUE THREAD

**Issue by sethkfman** [MetaMask/metamask-mobile#2516]
1. Connect CI (include all CI activities)
2. Generate Artifacts
3. Deploy to Test User Base (Connect to Slack)

**Comment by mobularay** [comment]
@sethkfman if this issue is currently being blocked by another issue, why is it in "Review/QA" column? Should it be in "In Progress" column? If it's in the correct column, then does it require dev review now, or QA review? 

Thanks so much :-)

**Comment by sethkfman** [comment]
@mobularay Provided a comment on the parent story. All Bitrise stories with dependencies are now in the In Review/QA column
